### PR TITLE
Test coverage increase

### PR DESCRIPTION
**Pull request by pozil**

### CODE REVIEW

File: src/modules/recipe/compositionContactSearch/__tests__/compositionContactSearch.test.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        searchInput.value = SEARCH_KEY;
+        searchInput.dispatchEvent(new CustomEvent('change'));
+
+        // Disable search throttling
+        jest.runAllTimers();
+
+        expect(findContactsMock).toHaveBeenCalledWith(SEARCH_KEY);
+    });
+
+    it('displays search results', () => {
```
Comment (muenzpraeger) [inline src/modules/recipe/compositionContactSearch/__tests__/compositionContactSearch.test.js]: I'd like to see overall alignment between the tests in `lwc-recipes` and `lwc-recipes-oss`.

File: src/modules/recipe/contactListItem/__tests__/contactListItem.test.js [hunk 2]
```diff
@@ -33,4 +33,25 @@ describe('recipe-contact-list-item', () => {
         const nameEl = element.shadowRoot.querySelector('p');
         expect(nameEl.textContent).toBe(CONTACT.Name);
     });
+
+    it('fires select event when clicked', () => {
```
Comment (muenzpraeger) [inline src/modules/recipe/contactListItem/__tests__/contactListItem.test.js]: I'd like to see overall alignment between the tests in `lwc-recipes` and `lwc-recipes-oss`.

File: src/modules/recipe/eventBubbling/__tests__/eventBubbling.test.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+        // Prevent data saved on mocks from leaking between tests
+        jest.clearAllMocks();
+    });
+
+    it('renders contact list from wire', () => {
```
Comment (muenzpraeger) [inline src/modules/recipe/eventBubbling/__tests__/eventBubbling.test.js]: I'd like to see overall alignment between the tests in `lwc-recipes` and `lwc-recipes-oss`.

File: src/modules/recipe/eventWithData/__tests__/eventWithData.test.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        Title: 'VP of Engineering',
+        Phone: '4152568563',
+        Email: 'amy@demo.net',
+        Picture: '/resources/images/demo/amy_taylor.jpg'
+    }
+];
+
+const getContactListWireAdapter = registerLdsTestWireAdapter(getContactList);
+
+describe('recipe-event-with-data', () => {
```
Comment (muenzpraeger) [inline src/modules/recipe/eventWithData/__tests__/eventWithData.test.js]: I'd like to see overall alignment between the tests in `lwc-recipes` and `lwc-recipes-oss`.

File: src/modules/recipe/compositionContactSearch/__tests__/compositionContactSearch.test.js (hunk 1 above)
Comment (pozil) [reply src/modules/recipe/compositionContactSearch/__tests__/compositionContactSearch.test.js]: done.

File: src/modules/recipe/contactListItem/__tests__/contactListItem.test.js (hunk 2 above)
Comment (pozil) [reply src/modules/recipe/contactListItem/__tests__/contactListItem.test.js]: done.

File: src/modules/recipe/contactListItemBubbling/__tests__/contactListItemBubbling.test.js [hunk 5]
```diff
@@ -33,4 +33,25 @@ describe('recipe-contact-list-item-bubbling', () => {
         const nameEl = element.shadowRoot.querySelector('p');
         expect(nameEl.textContent).toBe(CONTACT.Name);
     });
+
+    it('fires contactselect event when clicked', () => {
```
[the remark this answers is not in the document]
Comment (pozil) [reply src/modules/recipe/contactListItemBubbling/__tests__/contactListItemBubbling.test.js]: done.

File: src/modules/recipe/eventBubbling/__tests__/eventBubbling.test.js (hunk 3 above)
Comment (pozil) [reply src/modules/recipe/eventBubbling/__tests__/eventBubbling.test.js]: done.

File: src/modules/recipe/eventWithData/__tests__/eventWithData.test.js (hunk 4 above)
Comment (pozil) [reply src/modules/recipe/eventWithData/__tests__/eventWithData.test.js]: done.

File: src/modules/recipe/errorPanel/__tests__/errorPanel.test.js [hunk 6]
```diff
@@ -70,9 +74,27 @@ describe('recipe-error-panel', () => {
...
         return Promise.resolve().then(() => {
             const messageTexts = Array.from(
-                element.shadowRoot.querySelectorAll('p[class="error-message"]')
+                element.shadowRoot.querySelectorAll('p.error-message')
             ).map(errorMessage => (errorMessage = errorMessage.textContent));
             expect(messageTexts).toEqual(ERROR_MESSAGES_OUTPUT);
         });
     });
+
+    it('calls ldsUtils.reduceErrors', () => {
```
[the remark this answers is not in the document]
Comment (pozil) [reply src/modules/recipe/errorPanel/__tests__/errorPanel.test.js]: done.

**Pull request merged**